runbooks/ccf-client-settings-dynamodb-runbook.md

```diff
@@ -0,0 +1,104 @@
+# CCF Client Settings DynamoDB
```

Comment:
We can add a primary URL to raise the score a bit:
https://github.com/Financial-Times/upp-provisioners/tree/master/ccf-dynamodb

And add a link to the client setting API healtcheck for the Monitoring section, because it checks the DynamoDB health:
https://upp-prod-delivery-eu.upp.ft.com/__ccf-client-settings-api/__health

Reply:
If the idea is to increase the score only I would rather write NotApplicable for the Monitoring similar to what is done in the `UPP - Content Fluentd Kinesis Stream` runbook.